queries/leo/highlights.scm

```diff
@@ -0,0 +1,264 @@
+((identifier) @variable (#set! "priority" 95))
+
+[
+ "assert"
+ "assert_eq"
+ "assert_neq"
+ "block"
+ "console"
+ "const"
+ "constant"
+ "finalize"
+ "in"
+ "let"
+ "mapping"
+ "private"
+ "program"
+ "public"
+ "record"
+ "self"
+ "struct"
+] @keyword
+
+[
+ "transition"
+ "function"
+ "inline"
+] @keyword.function
+
+"import" @include
+
+"return" @keyword.return
+
+"for" @repeat
+
+[ 
+  "if"
+  "then"
+  "else"
+] @conditional
+
+[ ";" "," "::"] @punctuation.delimiter
+
+[
+"!"
+
+"&&"
+"||"
+
+"=="
+"!="
+
+"<"
+"<="
+">"
+">="
+
+"&"
+"|"
+"^"
+
+"<<"
+">>"
+
+"+"
+"-"
+"*"
+"/"
+"%"
+"**"
+
+"="
+
+"+="
+"-="
+"*="
+"/="
+"%="
+"**="
+
+"<<="
+">>="
+
+"&="
+"|="
+"^="
+
+"&&="
+"||="
+
+] @operator
+
+(comment) @comment
+
+(boolean_literal) @boolean
+
+(ternary_expression
+  (_)
+  (ternary_if) @conditional
+  (_)
+  (ternary_else) @conditional
+  (_))
```

Comment:
This can be changed to
```suggestion
[
  (ternary_if)
  (ternary_else)
] @conditional

Reply:
Fixed [f5cb905c](https://github.com/r001/nvim-treesitter/commit/f5cb905c780900f88f85a68223af628306b686b0)

Reply:
I don't see any changes from my end just yet. But it's nice that it's fixed